homeassistant/components/openweathermap/const.py

```diff
@@ -170,7 +170,7 @@
     },
     ATTR_API_PRESSURE: {
         SENSOR_NAME: "Pressure",
-        SENSOR_UNIT: PRESSURE_PA,
+        SENSOR_UNIT: PRESSURE_HPA,
```

Comment:
This is a breaking change; unit of measurement changed from Pa to hPa, which can affect things like InfluxDB.

Reply:
I am not aware of this kind of side effects. Breaking change checked.

Reply:
Thanks for adding it :+1: